modules/ivc_champva/app/form_mappings/vha_10_7959f_1.json.erb

```diff
@@ -1,13 +1,21 @@
 {
-  "form1[0].#subform[0].VetLastName[0]":          "<%= form.data.dig('veteran', 'full_name', 'last') %>",
+  "form1[0].#subform[0].VetLastName[0]":          "<%= form.data.dig('veteran', 'full_name', 'last') + ' ' + form.data.dig('veteran', 'full_name', 'suffix') %>",
   "form1[0].#subform[0].VetFirstName[0]":         "<%= form.data.dig('veteran', 'full_name', 'first') %>",
-  "form1[0].#subform[0].MiddleInitials[0]":       "<%= form.data.dig('veteran', 'full_name', 'middle') %>",
+  "form1[0].#subform[0].MiddleInitials[0]":       "<%= form.data.dig('veteran', 'full_name', 'middle')&.first %>",
   "form1[0].#subform[0].SocialSecurityNumber[0]": "<%= form.data.dig('veteran', 'ssn') %>",
   "form1[0].#subform[0].VAClaimFileNumber[0]":    "<%= form.data.dig('veteran', 'va_claim_number') %>",
   "form1[0].#subform[0].DateofBirth[0]":          "<%= form.data.dig('veteran', 'date_of_birth') %>",
-  "form1[0].#subform[0].PhysicalAddress[0]":      "<%= form.data.dig('veteran', 'physical_address', 'street_combined') + '\n' + form.data.dig('veteran', 'physical_address', 'city') + ', ' + form.data.dig('veteran', 'physical_address', 'state') + '\n' + form.data.dig('veteran', 'physical_address', 'postal_code') %>",
+  "form1[0].#subform[0].PhysicalAddress[0]":      "<%= form.data.dig('veteran', 'physical_address', 'street_combined')
+                                                    .split(/\n/).join('\n')\
```

Comment:
Since we don't know which streets the user may fill out (e.g., combinations of 1, 2, or 3), we combine them on the frontend into a single string with streetnames separated by `\n`. However, the `\n` from the frontend is not rendered in the PDF, so here we split on that `\n` and replace it in Ruby (which renders it properly in the PDF).